offline/utils.py

```diff
@@ -189,28 +185,6 @@ def load_xdf_data(file, with_stim=False):
     return raw, event_arr, event_id
 
 
-def create_config():
```

Comment:
Why is this function removed? I do not see it being added anywhere else? We still want to have this functionality I assume?

Reply:
It is removed because we now have human-readable configuration files.